src/datastreams.jl

```diff
@@ -101,10 +101,10 @@ Data.weakrefstrings(::Type{<:Statement}) = false
 Data.streamtypes(::Type{<:Statement}) = [Data.Row]
 
 function Data.streamto!(sink::Statement, ::Type{Data.Row}, row, row_num, col_num)
-    parameters = Vector{Parameter}(length(row))
+    parameters = Vector{Parameter}(undef, length(row))
 
     # this should change to be whatever custom pgtype conversion function we invent
-    map!(parameters, values(row)) do val
+    map!(parameters, collect(values(row))) do val
```

Comment:
Why `collect` now? What changed?

Reply:
Seems that `values` returns a `Tuple` rather than an array, which hits a `MethodError` from `map!`.

Reply:
We should make this a loop assigning to `parameters` instead then. Don't want to allocate an extra vector for each row.